txpool/queue_priced_test.go

```diff
@@ -232,15 +236,10 @@ func Test_maxPriceQueue(t *testing.T) {
 		t.Run(tt.name, func(t *testing.T) {
 			t.Parallel()
 
-			queue := &maxPriceQueue{
-				baseFee: tt.baseFee,
-				txs:     tt.unsorted,
-			}
-
-			sort.Sort(queue)
+			queue := newPricesQueue(tt.baseFee, tt.unsorted)
 
 			for _, tx := range tt.sorted {
```

Comment:
This could not work or be flaky as the order of range in a slice is not guaranteed.

Reply:
it is the slice, not the map https://stackoverflow.com/questions/45346641/slice-iteration-order-in-go
queue (or its encapsulated slice) is sorted by `heap.Init` and `tt.sorted` should be in expected order